src/PHYS/trczdf.f90

```diff
@@ -129,12 +133,20 @@ SUBROUTINE trczdf
         ztavg = 0.e0
 !! vertical slab
 
+        ! NOTE: kernel is too big, should be split
+        !$acc parallel loop gang vector default(present) async vector_length(32)
```

Comment:
we might want to think about clever ways to generate this kernel as it seems quite big, best performance on A100 was obtained with a vector length of 32 which isn't very high